package.json

```diff
@@ -4,23 +4,35 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "build": "ncc build src/setup-elixir.js",
-    "format": "prettier \"src/**/*.js\""
+    "build": "ncc build src/setup-elixir.js --no-cache --license licenses.txt",
+    "jslint": "eslint src/**/*.js && eslint __tests__/**/*.js",
+    "licenses": "yarn licenses generate-disclaimer > 3RD_PARTY_LICENSES",
+    "markdownlint": "markdownlint *.md",
+    "shellcheck": "shellcheck src/install-*",
+    "test": "node __tests__/setup-elixir.test.js",
+    "yamllint": "yamllint .github/workflows/**.yml && yamllint .*.yml && yamllint *.yml"
   },
   "husky": {
     "hooks": {
       "pre-commit": "npm run format -- --check && npm run build && git add dist/"
     }
   },
   "dependencies": {
-    "@actions/core": "^1.0.0",
-    "@actions/exec": "^1.0.0",
-    "@actions/tool-cache": "^1.1.0",
-    "semver": "^6.3.0"
+    "@actions/core": "1.1.0",
+    "@actions/exec": "1.0.1",
+    "semver": "7.3.4"
   },
   "devDependencies": {
-    "@zeit/ncc": "^0.22.1",
+    "@vercel/ncc": "0.27.0",
+    "eslint": "7.22.0",
+    "eslint-config-google": "0.14.0",
     "husky": "^4.2.5",
-    "prettier": "^2.0.5"
```

Comment:
@ericmj, I removed this. I'm not really sure it's useful. Lemme know.

Reply:
@paulo-ferraz-oliveira IMO it's useful if we need to format code, which I hope we would be.

Reply:
The way it was, IIRC, not only was the code not formatted (the one that was pushed) but the result was not taken into account. I don't mind adding something that actually does in-place code format replacement (which could be tested in CI), but as-was it was kinda limited.

Reply:
I've added this again. I'm afraid this increased the code to review by a bit, since it touched stuff that was previously unformatted. At the same time, it'll (most constantly) imply having to tweak the linter and prettier config. files at the same time.